src/components/pages/HomePage.js

```diff
@@ -1,41 +1,230 @@
-import { React,useState } from "react";
+import { React, useState } from "react";
+import { Link } from "react-router-dom";
 import Graph from "../reusable/Graph";
 import Navbar from "../reusable/NavBar";
 import Sidebar from "../reusable/SideBar";
 
 const HomePage = () => {
+  // Currently hardcoded but will eventually come from API
+  const [chartData, setChartData] = useState([
+    50, 41, 35, 51, 4, 62, 262, 91, 134,
+  ]);
 
-    // Currently hardcoded but will eventually come from API
-    const [chartData, setChartData] = useState([1000,50,1])
-    const [categories, setCategories] = useState([
-      "Jan",
-      "Feb",
-      "Mar",
-    ])
-  
-    /*
+  const [categories, setCategories] = useState([
+    "Jan",
+    "Feb",
+    "Mar",
+    "Apr",
+    "May",
+    "Jun",
+    "Jul",
+    "Aug",
+    "Sep",
+  ]);
+
+  const [selectedTabFilter, setSelectedTabFilter] = useState("investor");
+  const [selectedGraphFilter, setSelectedGraphFilter] = useState("Today");
+
+  const graphFilters = {
+    DAY: "Today",
+    FIVE: "Past 5 days",
+    MONTH: "Past month",
+    YEAR: "Past Year",
+  };
+
+  const tabFilters = {
+    INVESTOR: "investor",
+    JOB: "job",
+    history: "history",
+  };
+
+  const [stockData, setStockData] = useState([
+    {
+      symbol: "AAPL",
+      recentPrice: 150.23,
+      open: 140.0,
+      high: 162.5,
+      low: 126.5,
+      yearHigh: 167.0,
+      yearLow: 100.0,
+      percentChanged: 1.5,
+    },
+  ]);
+
+  const handleTabFilterSelection = (filter) => {
+    setSelectedTabFilter(filter);
+    // logic to update chart data based on selected filter
+  };
+
+  const handleGraphFilterSelection = (filter) => {
+    getGraphData(filter);
+    setSelectedGraphFilter(filter);
+    // logic to update chart data based on selected filter
+  };
+
+  /*
     Callback used to get more data based on the filter. Each time any of the buttons 
     are clicked this will be called to get more data. This will update the chart data which 
     will then re-render the graph
     */
-    const getData = (filter) => {
-      setChartData([])
-      setCategories([])
-      console.log(filter)
-  }
+  const getGraphData = (filter) => {
+    if (filter === "Past Year") {
+      setChartData([]);
+    } else {
+      setChartData(chartData.concat(chartData));
+      setCategories(categories.concat(categories));
+      setStockData(stockData);
+      console.log("Request filter: " + filter);
+    }
+  };
 
   return (
     <div className="bg-dark-gray overflow-x-auto overflow-y-auto">
       <Navbar />
       <div className="flex self-stretch">
         <Sidebar />
         <div className="sm:w-3/4 md:w-5/6 lg:w-7/8 p-5">
-          <div className="flex pt-10">
-            <h1 className="text-green font-bold text-5xl">Home</h1>
+          <div className="pt-10">
```

Comment:
"Your Assets can be broken off into its own component

Reply:
Why is this necessary @WillOrtiz7

Reply:
wait do you mean have all of the stock information in a separate component or just the title @WillOrtiz7